### ISSUE TITLE
Apollo branch needs to be updated to reflect current Apollo project.

### ISSUE THREAD

**Issue by pentatronix** [meteor/todos#242]
The current Apollo branch is a year old and only covers a very  simple integration of Apollo, specifically just the graphQL portion.  This branch should be updated to include the kinds of integrations found here:

http://dev.apollodata.com/core/meteor.html

Thanks!

**Comment by abernix** [comment]
Thanks for identifying this!  Is this something you'd be willing to help with, @pentatronix?

@hwillson @lorensr Is this something we should consider for todos?  Should we outline a general plan here?

**Comment by hwillson** [comment]
I think it's a great idea. There was some discussion about this a while back in https://github.com/meteor/todos/pull/167. People are already familiar with how todos works, so it would be great to demonstrate how Apollo can be used alongside Meteor, while leveraging the familiarity of todos. Relating to that PR, I had started a new version of todos a while back that demonstrated how to use Apollo Client + Apollo Server + MySQL with Meteor. Unfortunately I didn't get far before being pulled into other things, but this is definitely something I'd like to re-visit. That being said, I'm not sure when I'll get to it so if you're interested in helping with this @pentatronix (or if anyone else is interested), that would be great!

**Comment by lorensr** [comment]
Yeah I'd base it off the `react` branch and use `react-apollo` and `meteor/apollo`

**Comment by pentatronix** [comment]
I'm swamped on my end (And also not a react aficionado) so I probably won't be much use here.  

Considering the positioning of Meteor and Apollo, it would be good at least one current example of full usage kicking around.

**Comment by hwillson** [comment]
Quick update here - I'm currently working on this.
